forge/db/controllers/User.js

```diff
@@ -39,9 +44,14 @@ module.exports = {
     },
 
     resetPassword: async function (app, user, newPassword) {
-        // if (zxcvbn(newPassword.score < 3)) {
-        //     throw new Error('Password Too Weak')
-        // }
+        if (zxcvbn(newPassword).score < 3) {
```

Comment:
```suggestion
        if (zxcvbn(newPassword).score < 2) {
```

Everywhere else in the code the strength is tested to `< 2`

Reply:
I just uncommented your code ;)